dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -52,30 +47,17 @@ datanucleus-api-jdo/4.2.4//datanucleus-api-jdo-4.2.4.jar
 datanucleus-core/4.1.17//datanucleus-core-4.1.17.jar
 datanucleus-rdbms/4.1.19//datanucleus-rdbms-4.1.19.jar
 derby/10.14.2.0//derby-10.14.2.0.jar
-dropwizard-metrics-hadoop-metrics2-reporter/0.1.2//dropwizard-metrics-hadoop-metrics2-reporter-0.1.2.jar
+error_prone_annotations/2.5.1//error_prone_annotations-2.5.1.jar
+failureaccess/1.0.1//failureaccess-1.0.1.jar
 flatbuffers-java/1.9.0//flatbuffers-java-1.9.0.jar
 generex/1.0.2//generex-1.0.2.jar
-gson/2.2.4//gson-2.2.4.jar
-guava/14.0.1//guava-14.0.1.jar
+gson/2.8.6//gson-2.8.6.jar
```

Comment:
Do you know why this PR upgrade GSON in `Hadoop 3.2 + Hive 2.3` profile only?

Reply:
I think this is because `hadoop-common` has gson dependency.